Source/Core/UICommon/VideoUtils.cpp

```diff
@@ -7,6 +7,10 @@
 #include "Common/Assert.h"
 #include "VideoCommon/VideoConfig.h"
 
+#ifdef _WIN32
+#include <Windows.h>
```

Comment:
Is this also an "FIXME: Horrible hack" include?

Reply:
No, this file is just using some Windows API functions without including the header.